setup.py

```diff
@@ -28,6 +28,8 @@
 ]
 
 setup(
+    name="bigcode_eval",
+    python_requires='>=3.8',
```

Comment:
DS1000 benchmark requires python 3.7.10
```suggestion
    name="bigcode_eval",
    python_requires='>=3.7',
```